feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableGridItem.kt

```diff
@@ -78,7 +79,13 @@ fun TimetableGridItem(
                 .padding(all = TimetableGridItemDefaults.contentMargin)
                 .border(1.dp, LocalRoomTheme.current.primaryColor, shape)
                 .clip(shape)
-                .background(LocalRoomTheme.current.containerColor)
+                .background(
+                    if (isBookmarked) {
+                        LocalRoomTheme.current.primaryColor.copy(alpha = 0.3f)
+                    } else {
+                        LocalRoomTheme.current.containerColor
+                    },
+                )
```

Comment:
It's inconvenient if you can't tell whether something is bookmarked or not, so this PR also dynamically changes the background color based on the bookmark status.
If there's a better idea, I'll go with that.

Not bookmarked | Bookmarked.
:--: | :--:
<img src="https://github.com/user-attachments/assets/f5289f73-c9ed-4570-bf39-b0766997ba57" width="300" /> | <img src="https://github.com/user-attachments/assets/38f4675f-e5c3-4feb-9f7a-897bfcf33a05" width="300" />

Reply:
Ultimately, removing this part makes it impossible to tell at a glance which items are bookmarked and which aren't, which is extremely inconvenient.
However, as mentioned in the comments below, it seems there's a preference to avoid modifying the UI due to JVM-specific functionality, so implementing this feature has become impossible. :disappointed: 
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2325326310

Reply:
Understood. :saluting_face: 
I agree; it would be better to handle it as a separate issue.